app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,11 @@
 <resources>
-    <string name="common_ok">OK</string>
-    <string name="common_yes">Yes</string>
-    <string name="common_no">No</string>
-    <string name="common_cancel">Cancel</string>
-    <string name="common_delete">Delete</string>
-    <string name="common_remove">Remove</string>
+    <!-- Common strings, uncomment if you want to use them -->
+    <!--<string name="common_ok">OK</string>-->
+    <!--<string name="common_yes">Yes</string>-->
+    <!--<string name="common_no">No</string>-->
+    <!--<string name="common_cancel">Cancel</string>-->
+    <!--<string name="common_delete">Delete</string>-->
+    <!--<string name="common_remove">Remove</string>-->
 
```

Comment:
The reason these are added here is because they already used in almost every project so it's better to create them here instead of having people create and organize them every time. I would prefer to keep them uncommented, otherwise people wouldn't know that they even exist. I think it's fine to have a few unused resource warnings here since presumably they would end up being used in the actual projects.